### ISSUE TITLE
Service example

### ISSUE THREAD

**Issue by IliaEremin** [fs/android-examples#12]
Привести пример работы сервиса, который запускается, по событию появления интернета и делает какую-то работу. 
Если в этом время приложение активно, то оно должно отображать, что сервис запущен, начал делать работу, выполнил эту работу и доставить результат на экран. Если приложение не запущено, то работа просто выполняется.

**Comment by IliaEremin** [comment]
@RuslanZakirov @RamilGabdrakhmanov cc

**Comment by RamilGabdrakhmanov** [comment]
@IlyaEremin  Да, полезная штука. Принципиально, чтобы был сервис? На сколько я помню чтобы отловить событие появление интернета достаточно просто ресивера аля "изменилось соединение"

**Comment by IliaEremin** [comment]
@RamilGabdrakhmanov Как раз хотел написать, что не принципиально, но если операция long-running, то вроде как рекомендуется использовать сервис. 
Было бы еще здорово описать про то как соотносится application и service. Чтобы понимать когда он дохнет и когда что доступно.
И в чем отличие сервиса от операции в бекграунд потоке. В общем типа детальный разбор сервиса с практической стороны.
